### PR TITLE
Add Configuration files

### PR DESCRIPTION
**Pull request by MuhammadMaazFaisal**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,16 +1,24 @@
 {
-    "name": "laravel/laravel",
+    "name": "ibexoft/ibexpenses",
```
Comment (mjawaids) [inline composer.json]: Convert it to 'ibexoft/ibex-wallet'

File: README.md [hunk 2]
```diff
@@ -1,66 +1,6 @@
-<p align="center"><a href="https://laravel.com" target="_blank"><img src="https://raw.githubusercontent.com/laravel/art/master/logo-lockup/5%20SVG/2%20CMYK/1%20Full%20Color/laravel-logolockup-cmyk-red.svg" width="400" alt="Laravel Logo"></a></p>
+# Ibexpenses
```
Comment (mjawaids) [inline README.md]: Convert the name to Ibex Wallet (ibex-wallet) everywhere you see.

File: README.md [hunk 3]
```diff
@@ -1,66 +1,6 @@
...
+[![Open Source Helpers](https://www.codetriage.com/mjawaids/ibexpenses/badges/users.svg)](https://www.codetriage.com/mjawaids/ibexpenses)
 
-<p align="center">
-<a href="https://github.com/laravel/framework/actions"><img src="https://github.com/laravel/framework/workflows/tests/badge.svg" alt="Build Status"></a>
-<a href="https://packagist.org/packages/laravel/framework"><img src="https://img.shields.io/packagist/dt/laravel/framework" alt="Total Downloads"></a>
-<a href="https://packagist.org/packages/laravel/framework"><img src="https://img.shields.io/packagist/v/laravel/framework" alt="Latest Stable Version"></a>
-<a href="https://packagist.org/packages/laravel/framework"><img src="https://img.shields.io/packagist/l/laravel/framework" alt="License"></a>
-</p>
+(UNDER ACTIVE DEVELOPMENT. NOT EVEN IN 'BETA')
```
Comment (mjawaids) [inline README.md:4]: If code build/setup steps can be added would be great.

File: composer.json [hunk 4]
```diff
@@ -1,16 +1,24 @@
...
     "type": "project",
-    "description": "The skeleton application for the Laravel framework.",
-    "keywords": ["laravel", "framework"],
+    "description": "Ibexpenses - Manage your personal expenses",
```
Comment (mjawaids) [inline composer.json]: Change name to Ibex Wallet

**Review by mjawaids — CHANGES_REQUESTED** [review]

**Comment by MuhammadMaazFaisal** [comment]
@mjawaids Please take a look at this PR. I updated the package files, but I still can not figure out why the check keeps failing. package files with the previous version, exactly as you instructed, but it is still not functioning. I tried running it locally, but it functions properly there.

**Comment by mjawaids** [comment]
> @mjawaids Please take a look at this PR. I updated the package files, but I still can not figure out why the check keeps failing. package files with the previous version, exactly as you instructed, but it is still not functioning. I tried running it locally, but it functions properly there.

You can check the build error at https://github.com/Ibexoft/ibex-wallet/actions/runs/11257729571/job/31302658302?pr=79

According to the error `Your lock file does not contain a compatible set of packages. Please run composer update.`

You may need to update and then push both composer and lock files to github to pass the build.

**Comment by MuhammadMaazFaisal** [comment]
> > @mjawaids Please take a look at this PR. I updated the package files, but I still can not figure out why the check keeps failing. package files with the previous version, exactly as you instructed, but it is still not functioning. I tried running it locally, but it functions properly there.
> 
> You can check the build error at https://github.com/Ibexoft/ibex-wallet/actions/runs/11257729571/job/31302658302?pr=79
> 
> According to the error `Your lock file does not contain a compatible set of packages. Please run composer update.`
> 
> You may need to update and then push both composer and lock files to github to pass the build.

@mjawaids I have already done that and tried it again, but it is not working.

**Comment by mjawaids** [comment]
@MuhammadMaazFaisal ok I've fixed that composer error. Found the solution here - https://github.com/actions/starter-workflows/issues/747#issuecomment-747221689

Now it is giving error on unit tests. I hope you can now proceed with this and fix the remaining. If you get stuck let me know.

**Review by mjawaids — APPROVED** [review]

**Pull request merged**